### PR TITLE
People/sara/fixing const issues

### PR DESCRIPTION
**Pull request by sararathje**
@chzmnky @enochtsang Please Review.

This reverts the changes that @enochtsang made. I just took out the `let` and `const`, and replaced with `var`. I tried just adding a `use strict`, but I think scope was affected, and broke database declarations elsewhere? Because master is currently broken, I wanted to implement the temp fix, and then go back and replace with `let` and `const` when we have a working master. 
I also fixed tabbing issues, which is why it looks like an entirely new file.

### CODE REVIEW

**Comment by sararathje** [comment]
Closing, as I fixed the solution, and will put in a PR for that instead of reverting back.

**Comment by sararathje** [comment]
Realized I could just re-commit here instead. Re-opening.

File: server/index.js [hunk 1]
```diff
@@ -1,158 +1,163 @@
...
+      serverUtils.getAllServerInformation(servers[i], updateServer);
+    }
+  });
+}
+
+// I don't know that this is the best way to use the mongo db connection
+// but we basically just store the database reference and grab the collection
+// every time that we need it.
+// Once the connection is established we update everything every 15 seconds.
+MongoClient.connect("mongodb://10.2.32.84:27017/statusboardDb", function(err, db) {
```
Comment (chzmnky) [inline server/index.js]: Suggest moving the mongoDB URL to a variable at the top of the page.

File: server/index.js [hunk 2]
```diff
@@ -1,158 +1,163 @@
...
+  MongoClient = require('mongodb').MongoClient,
+  serverUtils = require('./serverUtils');
+
+let collection,
+  database,
+  statusboard,
+  handleResponseCallback;
+  
+function updateServer(server) {
+  database.collection("servers").updateOne(
```
Comment (chzmnky) [inline server/index.js]: Use single quotes for consistency.  I also see the "servers" collection as a string being passed around a lot - you may consider making this a constant to ease refactoring.

File: server/index.js [hunk 3]
```diff
@@ -1,158 +1,163 @@
...
+    {name: server.name},
+    {$set: server}
+  );
+}
+
+function updateServers() {
+  database.collection("servers").find().toArray(function(err, servers) {
```
Comment (chzmnky) [inline server/index.js]: Single quotes.

File: server/index.js (hunk 1 above)
Comment (chzmnky) [inline server/index.js]: Single quotes.

File: server/index.js [hunk 4]
```diff
@@ -1,158 +1,163 @@
...
+
+  database = db;
+
+  updateServers();
+  setInterval(updateServers, 15000);
+});
+
+app.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
```
Comment (chzmnky) [inline server/index.js]: Single quotes.

File: server/index.js [hunk 5]
```diff
@@ -1,158 +1,163 @@
...
+  next();
+});
+
+app.post('/server/new', function(req, res) {
+  var data = JSON.parse(req.query.data);
+
+  // Insert
+  if (data.name !== undefined) {
+    database.collection("servers").insertOne(data, function(err, r) {
```
Comment (chzmnky) [inline server/index.js]: Single quotes

File: server/index.js [hunk 6]
```diff
@@ -1,158 +1,163 @@
...
+  }
+});
+
+app.post('/server/update', function(req, res) {
+  var data = JSON.parse(req.query.data);
+
+  // Update
+  if (data.name !== undefined) {
+    // find our server and update it with the new data
+    database.collection("servers").updateOne(
```
Comment (chzmnky) [inline server/index.js]: Single quotes

File: server/index.js [hunk 7]
```diff
@@ -1,158 +1,163 @@
...
+  }
+});
+
+app.post('/server/delete', function(req, res) {
+  var data = JSON.parse(req.query.data);
+
+  // Update
+  if (data.name !== undefined) {
+    // find our server and update it with the new data
+    database.collection("servers").findOneAndDelete(
```
Comment (chzmnky) [inline server/index.js]: Single quotes

File: server/index.js [hunk 8]
```diff
@@ -1,158 +1,163 @@
...
+        return;
+      }
+    );
+  } else {
+    res.status(400).send('Data did not have a server name.');
+  }
+});
+
+app.get('/server', function(req, res) {
+  database.collection("servers").findOne({name: req.query.name}, function(err, r) {
```
Comment (chzmnky) [inline server/index.js]: Single quotes.

File: server/index.js [hunk 9]
```diff
@@ -1,158 +1,163 @@
...
+    }
+
+    res.status(200).send(JSON.stringify(r));
+
+    return;
+  });
+});
+
+app.get('/servers', function(req, res) {
+  database.collection("servers").find().toArray(function(err, servers) {
```
Comment (chzmnky) [inline server/index.js]: Single quotes.

File: server/index.js [hunk 10]
```diff
@@ -1,158 +1,163 @@
...
+// for development purposes.
+// app.post('/servers/delete', function(req, res) {
+//  database.collection("servers").removeMany();
+
+//  res.status(200).send();
+
+//  return;
+// });
+
+statusboard = app.listen(8081, function() {
```
Comment (chzmnky) [inline server/index.js]: Service port would be best stored as a constant, assigned at the top of the page.

File: server/index.js [hunk 11]
```diff
@@ -1,158 +1,165 @@
...
+      servicePort = 8081,
+      mongoUrl = 'mongodb://10.2.32.84:27017/statusboardDb';
+
+let collection,
+    database,
+    statusboard,
+    handleResponseCallback;
+  
+function updateServer(server) {
+  database.collection("servers").updateOne(
```
Comment (chzmnky) [inline server/index.js]: I missed one - single quotes pleeze.

**Pull request merged**